src/apps/external_executor/grpc_status.h

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache 2.0 License.
+#pragma once
+
+#include "status.pb.h"
+
+#include <optional>
+#include <string>
+
+// Mapping to HTTP errors as per
+// https://github.com/googleapis/googleapis/blob/master/google/rpc/code.proto
+#define GRPC_STATUS_MAP(XX) \
+  XX(0, OK, "Ok", 200) \
```

Comment:
The last value corresponds to the equivalent HTTP status but is not yet used. We can use it when we add generic support for gRPC errors at the framework level.